.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+**Describe the change**
```

Comment:
I would recommend putting this file in `.github/PULL_REQUEST_TEMPLATE/pull_request_template.md` to be consistent with the ISSUE_TEMPLATE.